SQL/Archive/17.0/2016-08-19-IssueTracker.sql

```diff
@@ -0,0 +1,106 @@
+-- Add issues tab to the Loris Menu
+INSERT INTO `LorisMenu` (`ID`, `Parent`, `Label`, `Link`, `Visible`, `OrderNumber`)
+VALUES (61, 5, 'Issue Tracker', 'issue_tracker/', NULL, 8);
```

Comment:
Everything has been working without LorisMenuPermissions - I've never touched that table. But I'll add it!